tests/acceptance/features/apiDownloads/download.feature

```diff
@@ -60,7 +60,15 @@ Feature: Download file in project space
       | Brian |
 
 
-  Scenario: user with role viewer cannot get the old version of the file in the project space
+  Scenario: user with role viewer cannot download the old version of the file in the project space
     Given user "Alice" has uploaded a file inside space "download file" with content "new content" to "file.txt"
-    When user "Bob" tries to get version of the file "file.txt" with the index "1" of the space "download file" using the WebDAV API
+    And user "Alice" has uploaded a file inside space "download file" with content "newest content" to "file.txt"
+    When user "Bob" tries to download version of the file "file.txt" with the index "1" of the space "download file" using the WebDAV API
```

Comment:
what's the difference between
`When user "Bob" tries to download version of the file "file.txt" with the index "1" of the space "download file" using the WebDAV API`
and
`When user "Bob" tries to get version of the file "file.txt" with the index 1 of the space "download file" using the WebDAV API`

Reply:
@saw-jan 
In the above When step, user is trying to download a version of a file with given index "1" and the index value is being used in downloadVersion() function. But, in another When, user is trying to get the version but in userTriesToDownloadFileVersions() function, and previously the index paramater was unused. That's why, "1" was changed to 1 just for the step to be understandable.

Reply:
hmm, this creates confusion. because I would think `get` and `download` are synonymous. and both steps should behave same.
Is there any difference in the behavior?

Reply:
which file version does `When user "Bob" tries to get version of the file "file.txt" with the index 1 of the space "download file" using the WebDAV API` return?

Reply:
From my knowledge, get -> tries to get the version of a file and download -> downloads the file with the specified index.

Reply:
> which file version does `When user "Bob" tries to get version of the file "file.txt" with the index 1 of the space "download file" using the WebDAV API` return?

None, because "Bob" doesn't have the access/permissions. Hence, the status code will be 403.

Reply:
now I got what the step is trying to do :+1:

Reply:
It seemed clearer to write `tries to download version of the file`
"download" is an explicit thing that readers of the step should understand in this kind of file-sync-and-share context. 
"get" was less specific. The old step was actually "getting" the list of versions of the file that are available on the server, and that is why the "index" parameter was not being used.

Reply:
Yeah, "download" was much more specific and explicit than "get" that's why there was a little confusion.